__tests__/api/league/route_integration.js

```diff
@@ -36,7 +45,16 @@ describe("POST", () => {
     it("should return a 401 when auth token is malformed (parts are not base64 encoded json)", async () => {
         // Arrange
         const request = {
-            json: async () => { return { token: "someTokenHeader.someTokenBody.someTokenSig" } }
+            cookies: {
+                get: () => { return {
+                    session: "someTokenHeader.someTokenBody.someTokenSig"
+                } }
+            },
+            json: async () => { 
+                return { 
+                    token: "someTokenHeader.someTokenBody.someTokenSig" 
+                }
```

Comment:
Does this still need to return anything?

```suggestion
                return { }
```